ngap/dispatcher.go

```diff
@@ -193,13 +204,17 @@ func DispatchNgapMsg(ran *context.AmfRan, pdu *ngapType.NGAPPDU) {
 }
 
 func HandleSCTPNotification(conn net.Conn, notification sctp.Notification) {
-	amfSelf := context.AMF_Self()
+    fmt.Println("Conn %v notification %v ", conn, notification)
+}
+func HandleSCTPNotificationSctp(remoteAddr string, notification sctp.Notification) {
 
-	logger.NgapLog.Infof("Handle SCTP Notification[addr: %+v]", conn.RemoteAddr())
+	logger.NgapLog.Infof("Handle SCTP Notification[addr: %+v]", remoteAddr)
 
+/*
```

Comment:
Need to uncomment this code ?
 - How handle Notification Sctp is called ?

Reply:
SCTP POD should inform if gNB is going down and then AMF POD can invoke this notification function.